Various/tompad_obliques.lua

```diff
@@ -0,0 +1,137 @@
+-- @description obliques
+-- @author tompad
+-- @version 1.0
+-- @provides tompad_obliques/obliques.lua
+-- @about
+--   # tompad_obliques
+--
+--   tompad_obliques is a reascript that randomly shows text
+--   from Oblique Strategies by Brian Eno and Peter Schmidt.
+--
+--   Any comments on coding, requests, bugs etc is welcome! PM me (tompad) on Reaper Forum (https://forum.cockos.com/member.php?u=19103).
+--
+--   Credits
+--   Oblique Strategies by Brian Eno and Peter Schmidt
+
+-- @description tompad_obliques - randomly shows text from Oblique Strategies
```

Comment:
There are two metadata headers in this main file. Only the first one will be used.